test/unit/auth/token-verifier.spec.ts

```diff
@@ -104,6 +104,17 @@ function mockFailedFetchPublicKeys(): nock.Scope {
     .replyWithError('message');
 }
 
+function createTokenVerifier(app: FirebaseApp, useEmulator: boolean): verifier.FirebaseTokenVerifier {
```

Comment:
nit: Please convert `useEmulator` to an options hash (e.g. `{useEmulator: true}`). The boolean literal arguments below are pretty hard to read.